modules/apps/ccv/parent/types/utils.go

```diff
@@ -0,0 +1,10 @@
+package types
+
+import "time"
+
+// TimeDelay is 4 weeks to ensure channel doesn't close on timeout
+const TimeDelay = 4 * 7 * 24 * time.Hour
+
+func GetTimeoutTimestamp(currentTime time.Time) time.Time {
+	return currentTime.Add(TimeDelay)
+}
```

Comment:
Equal to the un-bonding time, we should add an estimate for the relayer's delay.

Reply:
I would expect that any time delay on normal relayer to be negligible with respect to unbonding period. Even if we give relayer 24 hrs, 28 days and 29 days doesn't matter so much